city_scrapers/spiders/chi_ssa_22.py

```diff
@@ -0,0 +1,124 @@
+                    '$entry_cnt]',
+                    entry_cnt=entry_cnt
+                ):
+                    for item in ul.xpath('./li'):
+
+                        item_str = ' '.join(item.xpath('./text()').extract_first().split(' ')[0:2])
+                        # logging.debug(item_str)
+                        start = self._parse_start(item_str, year)
+                        date = start.date()
+
+                        meetings[date] = {'start': start, 'end': None, 'links': []}
+
+                        for a in item.xpath('./a'):
+
+                            item_str = a.xpath('./text()').extract_first()
+                            item_links = a.xpath('@href').extract()
+                            # logging.debug(item_str)
+                            meetings[date]['links'].extend(self._parse_links(item_links, item_str))
+
+        # Create the meeting objects
+        for key, item in meetings.items():
+
+            if item['start'] < last_year and not self.settings.getbool("CITY_SCRAPERS_ARCHIVE"):
+                continue
+
+            meeting = Meeting(
+                title='Commission',
+                description=unicodedata.normalize("NFKD", general_desc),
+                classification=COMMISSION,
+                start=item['start'],
+                end=item['end'],
+                time_notes='9:30am or 3:45pm (Please check our '
+                'Monthly Newsletter for more information)',
+                all_day=False,
+                location=self.location,
+                links=item['links'],
+                source=response.url,
+            )
+
+            meeting['status'] = self._get_status(meeting)
+            meeting['id'] = self._get_id(meeting)
+            yield meeting
+
+    def _parse_links(self, items, item_str):
+        """Parse documents if available on previous board meeting pages"""
+        documents = []
+        for url in items:
+            if url:
+                documents.append(self._build_link_dict(url, item_str))
+
+        return documents
+
+    @staticmethod
+    def _build_link_dict(url, item_str):
+        """Determine link type based on link text content"""
+        if 'agenda' in item_str.lower():
+            return {'href': url, 'title': 'Agenda'}
+        elif 'minutes' in item_str.lower():
+            return {'href': url, 'title': 'Minutes'}
+        else:
+            return {'href': url, 'title': 'Link'}
+
+    @staticmethod
+    def _validate_location(text):
+        """Parse or generate location."""
+        if "5153" not in text:
+            raise ValueError("Meeting location has changed")
+
+    @staticmethod
+    def _parse_date(raw_date, year):
+        """Parse date."""
+        if raw_date:
+            return dateutil.parser.parse(year + ' ' + raw_date)
+
+    def _parse_start(self, item_str, year):
+        """Parse start datetime."""
+        start = self._parse_date(item_str, year)
```

Comment:
If we aren't sure of the start time it's probably safer to leave it as midnight. This means we can also remove the `time` property from the class

Reply:
OK, thanks for the hint.